docs/developer/discounts.mdx

```diff
@@ -0,0 +1,815 @@
+For instance, if a product is priced at **$9** and has a **10%** discount, it will be available for **$8.1**.
+The discount will be visible on the line prices. To calculate the total applied discount,
+subtract the `line.undiscountedTotalPrice` from the `line.totalPrice`.
+To calculate the discount on one unit, subtract the `line.undiscountedUnitPrice` from the `line.unitPrice`.
+
+:::info
+The sale discount is not visible on `checkout.discount.amount`.
+:::
+
+Consider the following example: after adding a product on sale to the checkout,
+the response will include the discounted price of the product.
+
+```graphql
+mutation {
+  checkoutLinesAdd(
+    checkoutId: "Q2hlY2tvdXQ6NzMwMzkwMmItZGRhOC00MzU3LTk1YTAtNjRiODNiNTllMmUy"
+    lines: { quantity: 1, variantId: "UHJvZHVjdFZhcmlhbnQ6Mzcx" }
+  ) {
+    errors {
+      message
+    }
+    checkout {
+      discount {
+        amount
+      }
+      token
+      lines {
+        quantity
+        variant {
+          name
+        }
+        totalPrice {
+          gross {
+            amount
+          }
+          net {
+            amount
+          }
+        }
+        undiscountedTotalPrice {
+          amount
+        }
+        unitPrice {
+          gross {
+            amount
+          }
+          net {
+            amount
+          }
+        }
+        undiscountedUnitPrice {
+          amount
+        }
+      }
+    }
+  }
+}
+```
+
+Please check the line prices in the following response.
+The line.totalPrice is **8.1**, and the `line.undiscountedTotalPrice` is **9.0**.
+
+```json {7,16-37}
+{
+  "data": {
+    "checkoutLinesAdd": {
+      "errors": [],
+      "checkout": {
+        "discount": {
+          "amount": 0.0
+        },
+        "token": "7303902b-dda8-4357-95a0-64b83b59e2e2",
+        "lines": [
+          {
+            "quantity": 1,
+            "variant": {
+              "name": "UHJvZHVjdFZhcmlhbnQ6Mzcx"
+            },
+            "totalPrice": {
+              "gross": {
+                "amount": 8.1
+              },
+              "net": {
+                "amount": 8.1
+              }
+            },
+            "undiscountedTotalPrice": {
+              "amount": 9.0
+            },
+            "unitPrice": {
+              "gross": {
+                "amount": 8.1
+              },
+              "net": {
+                "amount": 8.1
+              }
+            },
+            "undiscountedUnitPrice": {
+              "amount": 9.0
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Voucher
+
+There are three types of vouchers:
+
+- Fixed amount: reduces the price by a specified value.
+- Percentage: reduces the price by a specified percentage value.
+- Shipping: reduces the shipping price.
+
+The voucher can be applied to all checkout products, or only to specified ones.
+
+There is also an option to apply the discount only to the cheapest eligible product.
```

Comment:
```
There is also an option to apply the discount only to the cheapest eligible product.
```

What does "eligible" mean here?

Reply:
I mean the product that is included in the voucher. But maybe it's not needed here?